src/Command/DrupalUpdate/DrupalUpdateCommand.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Acquia\Cli\Command\CommandBase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DrupalUpdateCommand extends  CommandBase {
+  /**
+   * @var string
+   */
+  protected static $defaultName = 'app:update-d7-packages';
+  /**
+   * @var string
+   */
+  private string $drupalProjectCwd;
+  /**
+   * @var mixed
+   */
+  private $drupalCoreVersion;
+  /**
+   * @var UpdateDrupalPackage
+   */
+  private UpdateDrupalPackage $updateDrupalPackage;
+
+  /**
+   * @var CheckUpdatesAvailable
+   */
+  private CheckUpdatesAvailable $checkUpdatesAvailable;
+
+  /**
+   * @return CheckUpdatesAvailable
+   */
+  public function getCheckUpdatesAvailable(): CheckUpdatesAvailable {
+    return $this->checkUpdatesAvailable;
+  }
+
+  /**
+   * @param CheckUpdatesAvailable $checkUpdatesAvailable
+   */
+  public function setCheckUpdatesAvailable(CheckUpdatesAvailable $checkUpdatesAvailable): void {
+    $this->checkUpdatesAvailable = $checkUpdatesAvailable;
+  }
+
+  /**
+   * @return UpdateDrupalPackage
+   */
+  public function getUpdateDrupalPackage(): UpdateDrupalPackage {
+    return $this->updateDrupalPackage;
+  }
+
+  /**
+   * @param UpdateDrupalPackage $updateDrupalPackage
+   */
+  public function setUpdateDrupalPackage(UpdateDrupalPackage $updateDrupalPackage): void {
+    $this->updateDrupalPackage = $updateDrupalPackage;
+  }
+
+  /**
+   * @return mixed
+   */
+  public function getDrupalCoreVersion():  mixed {
+    return $this->drupalCoreVersion;
+  }
+
+  /**
+   * @param mixed $drupalCoreVersion
+   */
+  public function setDrupalCoreVersion(mixed $drupalCoreVersion): void {
+    $this->drupalCoreVersion = $drupalCoreVersion;
+  }
+
+  /**
+   * @return string
+   */
+  public function getDrupalProjectCwd(): string {
+    return $this->drupalProjectCwd;
+  }
+
+  /**
+   * @param string $drupalProjectCwd
+   */
+  public function setDrupalProjectCwd(string $drupalProjectCwd): void {
+    $this->drupalProjectCwd = $drupalProjectCwd;
```

Comment:
This duplicates this existing repoRoot property, right?

Reply:
Yes, you are correct, let me update this.